packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -222,7 +243,7 @@ const ProductDetail = () => {
                     addToCart={(variant, quantity) => handleAddToCart(variant, quantity)}
                     addToWishlist={(_, quantity) => handleAddToWishlist(quantity)}
                     isProductLoading={isProductLoading}
-                    isCustomerProductListLoading={!wishlist.isInitialized}
+                    isCustomerProductListLoading={isWishlistLoading}
```

Comment:
nit, feels like this makes more sense to end users.
```suggestion
                    isWishlistLoading={isWishlistLoading}
```